### ISSUE TITLE
Low mAP scores during training on COCO: from 0.01 to 0.1 during 60 epochs

### ISSUE THREAD

**Issue by Dawars** [fizyr/keras-retinanet#1467]
I've been training the model with default parameters for 60 epoch now on COCO (2017) and I'm not getting the expected performance

The command I used: 
`python keras_retinanet/bin/train.py --tensorboard-dir ./logs_coco_vanilla --gpu 0 --steps 10000 --compute-val-loss --epochs 100  --batch-size 1 coco /workspace/datasets/coco`

1. The mAP scores start very low (0.01) and after 60 epochs only reach 0.1-0.2
2. At epoch 30 the gap between the train/val loss starts to widen which I take to be the beginning of overfitting.

![Screenshot from 2020-10-01 09-37-53](https://user-images.githubusercontent.com/1162980/94781268-e0d07600-03c9-11eb-8656-d449a577acd1.png)
![Screenshot from 2020-10-01 09-38-13](https://user-images.githubusercontent.com/1162980/94781276-e29a3980-03c9-11eb-895f-22fec013fc36.png)

I'm training on a GTX 1080 Ti.

Is this normal? If now, what could be the cause?

**Comment by pratibhashinde** [comment]
I am also getting low mAP. first epoch's  mAP is 0.0017. I am using ResNet50 backbone on my own dataset. I did not make any changes in the program. Is it normal or do I need to make some changes?
